src/main/resources/templates/db/migration/V3__Add_Users.sql

```diff
@@ -0,0 +1,33 @@
+create table cards (
```

Comment:
We already have this table created. We need to only put new db objects in each new migration